xlml/utils/name_format.py

```diff
@@ -56,4 +56,4 @@ def generate_gcs_file_location(benchmark_id: str) -> str:
     gsc file name with location
   """
   current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
-  return f"{gcs_bucket.BENCHMARK_OUTPUT_DIR}/{benchmark_id}-{current_datetime}/metric_report.jsonl"
+  return f"{gcs_bucket.BENCHMARK_OUTPUT_DIR}/{benchmark_id}-{current_datetime}/"
```

Comment:
Only include benchmark output dir here is not right. 

Can we combine this two as one output folder? https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/f65c4fb9d66ecf4e3 remove benchmark & xlml suffix

Reply:
Can you update the link in the comment? It's not pointing to the correct file.

Reply:
I see, add `OUTPUT_DIR = "gs://ml-auto-solutions/output/"`.